### PR TITLE
Remove wyvern-schemas package

### PR DESCRIPTION
**Pull request by JoshuaSchmidt-OpenSea**
Removing this package fixes known security issues with the version of `axios` used by the wyvern-schemas package

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -105,5 +106,12 @@
     "**/*.{ts,tsx,js,jsx}": [
       "eslint --cache --fix"
     ]
+  },
```
Comment (JoshuaSchmidt-OpenSea) [inline package.json:109]: 👀

File: src/sdk.ts [hunk 2]
```diff
@@ -608,7 +609,7 @@ export class OpenSeaSDK {
     amountInEth: number;
     accountAddress: string;
   }) {
-    const token = WyvernSchemas.tokens[this._networkName].canonicalWrappedEther;
+    const token = getCanonicalWrappedEther(this._networkName);
```
Comment (JoshuaSchmidt-OpenSea) [inline src/sdk.ts:612]: 👀

File: src/utils/tokens/index.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  address: string;
+}
+
+export interface NetworkTokens {
+  canonicalWrappedEther: Token;
+  otherTokens: Token[];
+}
+
+// TODO: Add Tests
+export const getCanonicalWrappedEther = function (network: Network): Token {
```
Comment (JoshuaSchmidt-OpenSea) [inline src/utils/tokens/index.ts:19]: 👀

File: src/utils/schemas/main/CryptoKitties/index.ts [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+  FunctionInputKind,
+  FunctionOutputKind,
+  StateMutability,
+} from "wyvern-js/lib/types";
+import { Schema } from "../../schema";
+import { EventInputKind } from "../../types";
+
+type CryptoKittiesType = string;
+
+export const CryptoKittiesSchema: Schema<CryptoKittiesType> = {
```
Comment (cterech) [inline src/utils/schemas/main/CryptoKitties/index.ts:13]: do we actually need this?

File: src/utils/schemas/main/CryptoPunks/index.ts [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+  FunctionOutputKind,
+  StateMutability,
+} from "wyvern-js/lib/types";
+import { Schema } from "../../schema";
+import { EventInputKind } from "../../types";
+
+type CryptoPunksType = string;
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export const CryptoPunksSchema: Schema<CryptoPunksType> = {
```
Comment (cterech) [inline src/utils/schemas/main/CryptoPunks/index.ts:13]: do we need this? what are these collection specific objects used for?

**Review by cterech — COMMENTED** [review]
few higher level comments
1. can we get rid of the collection specific definitions? i.e. why do we need a CryptoKitties schema? 
2. is it a lot of additional work to remove all of the references to Rinkeby? 
3. (this might be a dumb question) - instead of copying and pasting all of these object definitions, is there a way we can define our own minimal set of schemas and use those?

File: src/utils/schemas/main/CryptoKitties/index.ts (hunk 4 above)
Comment (JoshuaSchmidt-OpenSea) [reply src/utils/schemas/main/CryptoKitties/index.ts:13]: We would need this if someone creates an order for CyptoKitties

**Comment by JoshuaSchmidt-OpenSea** [comment]
> few higher level comments
> 
> 1. can we get rid of the collection specific definitions? i.e. why do we need a CryptoKitties schema?
> 2. is it a lot of additional work to remove all of the references to Rinkeby?
> 3. (this might be a dumb question) - instead of copying and pasting all of these object definitions, is there a way we can define our own minimal set of schemas and use those?

I think we will be able to remove CryptoPunks, Rinkeby and possible a set of the schemas and tokens. However, I would like to not make any changes to the logic in this PR since it is already massive just by removing the package.

File: package.json [hunk 6]
```diff
@@ -22,7 +22,7 @@
     "check-types": "tsc --noEmit --project tsconfig.json",
     "coverage-report": "nyc report --reporter=text-lcov | coveralls",
     "docs-build": "typedoc --out docs src/index.ts",
-    "eslint:check": "eslint . --max-warnings 0 --ext .js,.ts",
+    "eslint:check": "eslint . --fix --max-warnings 0 --ext .js,.ts",
```
Comment (montasaurus) [inline package.json:25]: Don't think we want this here--should fail instead of fixing. Could add another `eslint:fix` if you want

**Review by montasaurus — APPROVED** [review]

**Review by cterech — APPROVED** [review]

**Pull request merged**